[2022-06-29 21:46:30,751] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: SDG_Enrique_V0.postgreSQL_getTable 2022-06-29T21:46:28.658958+00:00 [queued]>
[2022-06-29 21:46:30,771] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: SDG_Enrique_V0.postgreSQL_getTable 2022-06-29T21:46:28.658958+00:00 [queued]>
[2022-06-29 21:46:30,772] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-06-29 21:46:30,773] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-06-29 21:46:30,775] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-06-29 21:46:30,787] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): postgreSQL_getTable> on 2022-06-29T21:46:28.658958+00:00
[2022-06-29 21:46:30,800] {standard_task_runner.py:52} INFO - Started process 2739 to run task
[2022-06-29 21:46:30,805] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'SDG_Enrique_V0', 'postgreSQL_getTable', '2022-06-29T21:46:28.658958+00:00', '--job-id', '1456', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/company_forecasting.py', '--cfg-path', '/tmp/tmpclqtiosu', '--error-file', '/tmp/tmp5s_k_97u']
[2022-06-29 21:46:30,809] {standard_task_runner.py:77} INFO - Job 1456: Subtask postgreSQL_getTable
[2022-06-29 21:46:30,863] {logging_mixin.py:104} INFO - Running <TaskInstance: SDG_Enrique_V0.postgreSQL_getTable 2022-06-29T21:46:28.658958+00:00 [running]> on host 3cf820b1df4c
[2022-06-29 21:46:30,952] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=enrique-liebana@outlook.com
AIRFLOW_CTX_DAG_OWNER=Enrique Liebana Pe√±a
AIRFLOW_CTX_DAG_ID=SDG_Enrique_V0
AIRFLOW_CTX_TASK_ID=postgreSQL_getTable
AIRFLOW_CTX_EXECUTION_DATE=2022-06-29T21:46:28.658958+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-29T21:46:28.658958+00:00
[2022-06-29 21:46:30,973] {base.py:74} INFO - Using connection to: id: postgres_conection. Host: host.docker.internal, Port: 5432, Schema: data_company, Login: airflow, Password: XXXXXXXX, extra: None
[2022-06-29 21:46:45,115] {company_forecasting.py:55} INFO - Backup Company data in : /raw_data/company.csv
[2022-06-29 21:46:45,117] {python.py:118} INFO - Done. Returned value was: None
[2022-06-29 21:46:45,128] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=SDG_Enrique_V0, task_id=postgreSQL_getTable, execution_date=20220629T214628, start_date=20220629T214630, end_date=20220629T214645
[2022-06-29 21:46:45,156] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-29 21:46:45,184] {local_task_job.py:146} INFO - Task exited with return code 0
